event_notification_examples/README.md

```diff
@@ -0,0 +1,14 @@
+# AWS Deadline Cloud Event Notification Examples
+
+## Introduction
+AWS Deadline Cloud sends events to customer's default event bus https://docs.aws.amazon.com/deadline-cloud/latest/userguide/monitoring-eventbridge.html
+This directory holds sample templates customers can use to setup notifications basis events received from Deadline Cloud
+through mechanisms like email or slack.
+
+## Setup Instructions
+Each template will have its own instructions on how to set up integrations, but in general they follow the below scheme:
```

Comment:
"Each template" -> "Each Cloud Formation Template"

-> Because many customers use Terraform and other tech too!